a2wsgi/wsgi.py

```diff
@@ -1,12 +1,19 @@
 import asyncio
 import collections
+import functools
 import os
 import sys
 import typing
 from concurrent.futures import ThreadPoolExecutor
 
 from .types import Environ, Message, Receive, Scope, Send, StartResponse, WSGIApp
 
+try:
```

Comment:
The new version of a2wsgi only supports above 3.7, so we can use contextvars directly.

Reply:
Aha, didn't know that.
I checked the [pyproject.toml](https://github.com/abersheeran/a2wsgi/blob/master/pyproject.toml#L11) which still includes 3.6.

Reply:
Ah, that was an oversight on my part.